File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.js
function solution(sides) {
    var answer = 0;
    sides.sort((a,b) => a-b);
    
    var a = sides[0]
    var b = sides[1]
    var c = b + 1
    
    // b가 최대일 때
    for (let i = 1; i < b + 1; i ++) {
        if (a + i > b) {
            answer += 1
        }
    }
    
    // c가 최대일 때
    while(a + b > c) {
        answer += 1
        c += 1
    }
    
    console.log(answer)
    
    return answer;
}

File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.py
def solution(sides):
    answer = 0
    sides.sort()
    a = sides[0]
    b = sides[1]
    c = b + 1
    
    # b가 최대일 때
    for i in range(1, b+1):
        if a + i > b:
            answer += 1
    
    # c가 최대일 때
    while a + b > c:
        answer += 1
        c += 1
    
    return answer